requirements.txt

```
@@ -21,3 +21,4 @@ fuzzywuzzy>=0.11,<=0.17
 pyelftools>=0.24,<=0.25
 git+https://github.com/armmbed/manifest-tool.git@v1.4.6
 icetea>=1.2.1,<1.3
+pyusb>=1.0.0
```

Could also add an upper bound on the API version?
```suggestion
pyusb>=1.0.0,<2.0.0
```